include/RAJA/scan.hxx

```diff
@@ -61,6 +61,15 @@
 namespace RAJA
 {
 
+namespace scan
+{
+template <typename ExecPolicy, typename T>
+struct defaults {
```

Comment:
@davidbeckingsale I created a `defaults` struct for scan which provides default binary operator and initial value given an execution policy and data type